announcements/2020-Week-33/apps.md

```diff
@@ -0,0 +1,15 @@
+# :zap: This Week in Non-Platform Releases (August 14, 2020):zap:
```

Comment:
@giantswarm/sig-product and maybe @stone-z :D

We're sending non-provider release notes (Optional Apps, Web Interface, Control Plane), etc. as a separate file and announcements. This reduce noise and duplicate content in release announcements. Not sure if you have better ideas on what to call this aside from "Non-Platform" 
* Other Releases?
* Non-Provider Releases?
* Provider Independent?